internal/boxcli/run.go

```diff
@@ -50,14 +47,6 @@ func runScriptCmd(cmd *cobra.Command, args []string, flags runCmdFlags) error {
 		return errors.WithStack(err)
 	}
 
-	// Validate script exists.
-	scripts := box.ListScripts()
-	sort.Slice(scripts, func(i, j int) bool { return scripts[i] < scripts[j] })
-	if script == "" || !slices.Contains(scripts, script) {
-		return errors.Errorf("no script found with name \"%s\". "+
-			"Here's a list of the existing scripts in devbox.json: %v", script, scripts)
-	}
```

Comment:
What happens when a user has a script in their devbox.json named "ls" and they run `devbox run ls` ?
Does the script alias take precedent over the command `ls`? or vice versa?

Reply:
The script takes precedence